Documentation/contributing/development/reviewers_committers/triage.rst

```diff
@@ -0,0 +1,153 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _triage:
+
+Triaging Issues
+===============
+
+The Triager role involves a one week rotation of looking at incoming
+bugs in Cilium and triaging them, as well as helping with merging and
+testing external contributions.
+
+Tasks
+-----
+Tasks are listed by order of importance.
+
+1. Merging and Pushing Contributions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the title indicates, this part involves looking at community contributions
+and seeing them through to completion. This involves approving workflows and
+calling ``/test``, pulling in code owners and, ultimately, merging the PR when
```

Comment:
For what it's worth, the actual expected people to do this work is, in order of responsibility:
1. The contributor creating the PR
2. Reviewers assigned to review the PR
3. Committers / maintainers who merge the PR
4. Triager to highlight that somehow the PR falls through the gaps because 1/2/3 did not achieve these steps.

Reply:
rewrote this section a bit to say they can help get things back on track, but aren't responsible. let me know if you think this is better

Reply:
Resolving as this is addressed in the proposed changes [below](https://github.com/cilium/cilium/pull/41871/files#r2431866486).